contrib/src/main/java/macrobase/analysis/stats/kde/TKDEEstimator.java

```diff
@@ -0,0 +1,88 @@
+package macrobase.analysis.stats.kde;
+
+import macrobase.analysis.stats.kde.grid.CompositeGrid;
+import macrobase.analysis.stats.kde.kernel.BandwidthSelector;
+import macrobase.analysis.stats.kde.kernel.Kernel;
+import macrobase.analysis.stats.kde.kernel.KernelFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class TKDEEstimator {
```

Comment:
What is an 'estimator'? Worth defining an interface?

at minimum, needs a comment